File: androidtoolkit/src/main/java/net/oschina/git/roland/androidtoolkit/common/StringUtils.java
package net.oschina.git.roland.androidtoolkit.common;

/**
 * Created by Roland on 2017/5/9.
 */

public class StringUtils {

    /**
     * 判断字符串是否为空
     * @param str 字符串
     * @return 是否为空
     */
    public static boolean isEmpty(String str) {
        return str == null || str.length() == 0 || str.trim().equals("") || str.trim().equalsIgnoreCase("null");
    }
}

File: androidtoolkit/src/main/java/net/oschina/git/roland/androidtoolkit/common/StringUtils.kt
package net.oschina.git.roland.androidtoolkit.common

/**
 * Created by Roland on 2017/5/9.
 */

object StringUtils {

    /**
     * 判断字符串是否为空
     * @param str 字符串
     * *
     * @return 是否为空
     */
    fun isEmpty(str: String?): Boolean {
        return str == null || str.isEmpty() || str.trim { it <= ' ' } == "" || str.trim { it <= ' ' }.equals("null", ignoreCase = true)
    }
}
